### ISSUE TITLE
The app started to crash after we started using BluetoothMedic tests

### ISSUE THREAD

**Issue by agap** [AltBeacon/android-beacon-library#672]
### Expected behavior
Does not crash

### Actual behavior
The app is crashing with the following stacktrace:
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke interface method 'boolean android.bluetooth.IBluetooth.isMultiAdvertisementSupported()' on a null object reference
       at android.bluetooth.BluetoothAdapter.isMultipleAdvertisementSupported(BluetoothAdapter.java:1201)
       at android.bluetooth.BluetoothAdapter.getBluetoothLeAdvertiser(BluetoothAdapter.java:631)
       at org.altbeacon.bluetooth.BluetoothMedic.runTransmitterTest(Unknown Source)
       at org.altbeacon.bluetooth.BluetoothTestJob$1.run(Unknown Source)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:145)
       at android.os.HandlerThread.run(HandlerThread.java:61)
```
### Steps to reproduce this behavior
Unknown
### Mobile device model and OS version
According to Crashlytics, 100% of crashes (on our app) happened on Samsung Galaxy S4, running Android 5.0.1

### Android Beacon Library version
2.13.1

**Comment by davidgyoung** [comment]
See also #668

**Comment by agap** [comment]
**Upd.**: The crash also seems to happen on Samsung Galaxy Note3, also running Android 5.x.

**Comment by davidgyoung** [comment]
@agap, I have a proposed fix in #675, but I cannot verify that it works without a reproducible test case.

Would you be willing to try it out in an app update to see if it eliminates these crashes?  If so, I will roll you an ad hoc binary release with instructions on using it with your app.

**Comment by davidgyoung** [comment]
#675 merged into master and released as 2.14
